themes/bootstrap3/templates/RecordDriver/EDS/cover.phtml

```diff
@@ -1,20 +1,16 @@
-<?php
-  // set $alt to empty when we have an icon ($cover === false), because we don't want to link the icons (it looks weird)
```

Comment:
Is it a problem that the icon special case is no longer handled here? Or is this no longer relevant? (I confess that I haven't tested or investigated yet -- I just noticed that this was removed and didn't appear to have any equivalent in the revised code).

Reply:
You are right, I missed that.

Reply:
I wonder if it would be slightly more clear to do something like:

```
$includeLinkInMarkup = $this->link && $cover !== false;
```

...and then use `$includeLinkInMarkup` in the if conditions on the open and close tags. This way, we don't have duplicated multi-part logic in multiple places, so we'll be less likely to get the code out of sync if we make future changes.